referrer-policy/generic/test-case.sub.js

```diff
@@ -74,14 +80,12 @@ function checkResult(scenario, expectation, result) {
     referrerUrlResolver[expectation](referrerSource);
 
   // Check the reported URL.
-  assert_equals(result.referrer,
-                expectedReferrerUrl,
-                "Reported Referrer URL is '" +
-                expectation + "'.");
-  assert_equals(result.headers.referer,
-                expectedReferrerUrl,
-                "Reported Referrer URL from HTTP header is '" +
-                expectedReferrerUrl + "'");
+  assert_in_array(result.referrer,
+                  expectedReferrerUrl,
```

Comment:
```suggestion
                  expectedReferrerUrls,
```

Same above and below, to make it clearer that this variable holds multiple possible expected values.

Reply:
Changed this to `possibleReferrerUrls` given it's not like we're expecting multiple URLs, we're expecting one of them.